AutoCollection/ServerRequestParser.ts

```diff
@@ -110,6 +106,10 @@ class ServerRequestParser extends RequestParser {
         return this.requestId;
     }
 
+    public getCorrelationContext() {
+        return this.correlationContext;
```

Comment:
Ah yes, sorry I thought this class had access to the correlation context and could fill it directly. It would make sense since that is part of the header parsing